### PR TITLE
[TW-1475] Share examples

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Added that you can also share examples.
* Clarified which elements have access to which flows.
* Updated section about sharing APIs, and added sections about sharing examples and requests.

[Beta site](https://tw-1475-share-examples.learning.postman-beta.com/docs/collaborating-in-postman/sharing/)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 1]
```diff
@@ -46,25 +46,25 @@ To share your work using workspaces, you need to be signed in to your [Postman a
...
+* To share a URL to a collection, collection folder, request, or example with an active environment selected, [choose an environment using the environment selector](/docs/sending-requests/environments/managing-environments/#switch-between-environments), copy the element's link to your clipboard, then send the link to team members you want to share with.
 
 ### Sharing using names, emails, or groups
 
-You can share collections, APIs, and environments directly with collaborators, including team members, groups, and external users.
+You can share collections, requests, examples, collection folders, APIs, Flows, and environments, directly with collaborators, including team members, groups, and external users.
 
-1. From the element you want to share, select <img alt="Share icon" src="https://assets.postman.com/postman-docs/icon-share.jpg#icon" width="16px"> **Share**.
-1. Select **With people**.
+1. From the element you want to share, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the element you want to share, then select **Share**.
```
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md:61]: Thanks for catching this!

File: src/pages/docs/collaborating-in-postman/sharing.md [hunk 2]
```diff
@@ -46,25 +46,25 @@ To share your work using workspaces, you need to be signed in to your [Postman a
...
 * Hover over the element's name in the overview page and select the link icon <img alt="Link icon" src="https://assets.postman.com/postman-docs/icon-workspace-link-v9.jpg#icon" width="18px"> to copy the element's URL to the clipboard.
-* You can also select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the element you want to share and select **Copy Link** to copy the element's URL to the clipboard.
-* To share a URL to a collection, folder, or request with an active environment selected, [choose an environment using the environment selector](/docs/sending-requests/environments/managing-environments/#switch-between-environments), copy the element's link to your clipboard, then send the link to team members you want to share with.
+* For requests and collection folders, you can also select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the element you want to share and select **Copy link** to copy the element's URL to the clipboard.
```
Comment (akshay-kulkarni-03) [inline src/pages/docs/collaborating-in-postman/sharing.md:54]: this is also applicable to examples right?

Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/sharing.md:54]: I'm not seeing this on beta for examples at the moment. I'll reach out to you via Slack.

**Review by akshay-kulkarni-03 — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**